whisper/whisperv5/envelope.go

```diff
@@ -172,6 +172,10 @@ func (e *Envelope) DecodeRLP(s *rlp.Stream) error {
 	if err != nil {
 		return err
 	}
+	return e.DecodeBytes(raw)
+}
+
+func (e *Envelope) DecodeBytes(raw []byte) error {
```

Comment:
There is no need to define this method. You can use rlp.DecodeBytes to decode an envelope from a byte slice.

```go
var e Envelope
rlp.DecodeBytes(raw, &e)
```